### DISCUSSION TITLE
Vercel blocking requests with payloads < 4.5MB of size

### DISCUSSION THREAD

**Discussion by Shiawaseu** [vercel/community#4151] [Help]
### Summary

- From the [docs](https://vercel.com/docs/functions/serverless-functions/runtimes#request-body-size)
> The maximum payload size for the request body or the response body of a Serverless Function is 4.5 MB. If a Serverless Function receives a payload in excess of the limit it will return an error 413: FUNCTION_PAYLOAD_TOO_LARGE.

Basically, I am trying to request my express.js endpoint which is hosted on Vercel with a Base-64 text body sizing **about 900 KB**, but I am still receiving a code 413.

I tried experimenting with smaller payloads and concluded that the real maximum in my case was **about 150 KB**.

Is there anything that I am missing or doing wrong? Please let me know.

**In Short:** Tried sending a payload with 990KB of size. Expected request to go through. Request was blocked for being too large instead.

![show](https://github.com/vercel/community/assets/73775954/1129f952-76f6-4f7c-a3b0-ef286703db99)

### Example

https://luadeck-server.vercel.app/beautify

### Steps to Reproduce

Send a payload sizing > 150 KB to `https://luadeck-server.vercel.app/beautify` [POST]
